src/components/buttons/PrimaryButton.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import PrimaryButton from "./PrimaryButton";
+
+export default {
+  title: "DroidconKe/Example/Button",
+  component: PrimaryButton,
+  parameters: {
+    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
```

Comment:
I didn't get how Storybook works till I checked out this docs.
I guess more comments on what does what will help?